Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -13,14 +13,40 @@ Microsoft Defender Advanced Threat Protection Get Machine Action Status
 2. Managing machines and performing actions on them.
 3. Blocking files and applications.
 
-## Configure Microsoft Defender Advanced Threat Protection on Cortex XSOAR
+## Required Permissions
+##### Microsoft Graph Permissions:
+* Directory.Read.All - Delegated
+* ThreatIndicators.Read.All - Application
+* ThreatIndicators.ReadWrite.OwnedBy - Application
+##### WindowsDefenderATP Permissions:
+* AdvancedQuery.Read.All - Application
+* Alert.Read.All - Application
+* Alert.ReadWrite.All - Application
+* Event.Write - Application
+* File.Read.All - Application
+* Ip.Read.All - Application
+* Machine.CollectForensics - Application
+* Machine.Isolate - Application
+* Machine.Offboard - Application
+* Machine.Read.All - Application
```

Comment:
we have a few permissions here we ask for both application and delegated? are we sure we need these?

Reply:
I'll arrange it better with the  Microsoft API reference